docs/scenario-dashboard.md

```diff
@@ -0,0 +1,34 @@
+# Batch Generator Dashboard
+
+monoDrive's Batch Generator Dashboard provides the client with visuals of custom configurations and result viewing for better testing.  The monoDrive simulator is configured from the Client. Everything about the simulator can be configured, from the material properties, to the vehicle properties, to the sensor properties. The dashboard is designed to step through each configuration with pre-built and customizable properties. Once all configurations are made, deploy and run every combination of the properties selected in the simulation. Results show the status of the test after it has run, and keeps different batches organized for review. The Dashboard can be used as a local app on your computer or through the cloud.
+
+<div class="img_container">
+  <video width=650px height=440px muted controls autoplay loop>
+    <source src="https://cdn.monodrive.io/cloud.mp4" type="video/mp4">
+  </video>
+</div>
+
+## Setup
+
+ 1. Download and run monoDrive Batch Generator Dashboard. *For access [contact us](https://www.monodrive.io/contact).*
+ 
+ 2. Saves results from the Unit Under Test (UUT) in the same directory that user specifies in app setup. 
+
+ 3. Play simulator with the UUT configured map and play UUT.
+ 
+ 4. Make selections and run batch. 
+     * Viewport Camera is for viewing the simulation in the simulator viewport.
+     * Refresh results and/or view later to see results after tests have run. 
```

Comment:
Should describe each section we are configuring. Don't think we need to describe the viewport camera specifically here. Maybe make the results monitoring a separate step